servers/rendering/renderer_rd/shaders/gi.glsl

```diff
@@ -7,6 +7,8 @@
 layout(local_size_x = 8, local_size_y = 8, local_size_z = 1) in;
 
 #define M_PI 3.141592
+#define M_TAU 6.283185
```

Comment:
I'm not sure the difference in precision is intentional here.

Maybe `M_PI`, `M_TAU`, and `M_E` should be defined in a `math_defs_inc.glsl` file that gets included everywhere?

Reply:
Yea, I suppose that would make more sense. I'll do that.